tests/std/tests/P0202R3_constexpr_algorithm_and_exchange/test.cpp

```diff
@@ -488,113 +464,113 @@ constexpr bool test_pop_heap_and_push_heap() {
     };
     assert(is_heap(begin(expectedPushed), end(expectedPushed)));
     assert(equal(begin(buff), end(buff), begin(expectedPushed), end(expectedPushed)));
-    return true;
 }
 
-constexpr bool test_permutations() {
-    int buff[] = {1, 2, 3, 4};

```

Comment:
&lt;meme>I don't always fail to run broken STL test cases, but when I do ...&lt;/meme>